src/pages/Store/StoreDetailPage/index.tsx

```diff
@@ -9,13 +9,15 @@ import { Portal } from 'components/common/Modal/PortalProvider';
 import useModalPortal from 'utils/hooks/useModalPortal';
 import useScrollToTop from 'utils/hooks/useScrollToTop';
 import useStoreDetail from './hooks/useStoreDetail';
+import useStoreMenus from './hooks/useStoreMenus';
 import styles from './StoreDetailPage.module.scss';
 
 function StoreDetailPage() {
   const params = useParams();
   const isMobile = useMediaQuery();
   const navigate = useNavigate();
-  const { storeDetail, storeDescription, storeMenus } = useStoreDetail(params.id);
+  const { storeDetail, storeDescription } = useStoreDetail(params.id ?? '');
+  const { storeMenus } = useStoreMenus(params.id ?? '');
```

Comment:
이 페이지의 라우팅이 파라미터로 id가 있어야만 하므로 id가 있다고 단언하는건 어떨까요?
```suggestion
  const { storeDetail, storeDescription } = useStoreDetail(params.id!);
  const { storeMenus } = useStoreMenus(params.id!);
```

바꾼다면 동시에 Number형으로도 바꾸고 싶다는 욕구가 동시에 드네요 ㅋㅋ